pkg/client/status_reporter.go

```diff
@@ -89,6 +90,7 @@ func (r *StatusReporter) SetInProgress() error {
 
 	conditions := newConditions(co.Status, r.version, time)
 	conditions.setCondition(v1.OperatorProgressing, v1.ConditionTrue, "Rolling out the stack.", time)
+	conditions.setCondition(v1.OperatorUpgradeable, v1.ConditionFalse, "", time)
```

Comment:
In the API docs for `OperatorUpgradeable`, it says we should set a reason for when this is false:

> and the message field should contain a human readable description of what the administrator should do to allow the operator to successfully update.

Reply:
great catch :+1: fixed